adopt-a-pup/Readme.MD

```diff
@@ -1,72 +1,51 @@
-oc adm policy add-scc-to-user anyuid -z maildev
-```
-MailDev is an open source email server testing application that allows you to spin up a fake email server for testing.
-Any email sent from the notification service to this pod will be visible in the UI described later.
- 
-`quay.io/redhattraining/maildev:latest` is a custom-built public image from https://github.com/maildev/maildev.
- 
-The original image can be found on docker hub at `https://hub.docker.com/r/maildev/maildev`.
+Note that this application is written for OpenShift Service Mesh.
+The project where you deploy this application should be managed by OSSM
+(i.e. be in the ServiceMeshMemberRoll resource).
 
-`oc new-app --name=email-service quay.io/redhattraining/maildev:latest`
-
-# Load data into Mongo
-There is dummy data in the adopt-a-pup/mongo-data directory that can be imported into the newly created mongo instance
-You must have mongodb-org installed (https://docs.mongodb.com/manual/tutorial/install-mongodb-on-red-hat/)
-
-```
-oc port-forward <mongo-pod-name> 27017:27017
-mongoimport --db=adopt-a-pup --collection=animals --username=developer --password=developer adopt-a-pup/mongo-data/animals.mongo
-mongoimport --db=adopt-a-pup --collection=shelters --username=developer --password=developer adopt-a-pup/mongo-data/shelters.mongo
-```
-# Swagger
-If you want to test any of the services running on OpenShift locally you must run `oc expose svc <service-name>` or port-forward the pod
-Then run `oc get routes` and use the route(s) in the response below
-
-`<ocp-host>/swagger-ui.html` for all of them
-
-# There are now two shelters each with 2 animals created.
-
-Denver Doggos: e038ae3c-592f-403e-9233-4b6eeab30e3c
-    - Harvey: e22d494c-c2be-4d32-bceb-ec675fd5540a
-    - Gus: d52a8d58-9024-49dd-92b6-d443c6049ffe
-Minneapolis Mutts: 6a432062-96a4-4a66-b888-1ab7225e6b2c
-    - Theo: b62977ad-fe79-4480-a550-06f717923017
-    - Winston: aac7ea0a-2374-4d4b-8d3a-71e4f896e751
+1. Create a project.
+    ```
+    oc new-project comprehensive-review
+    ```
     
-Once all the data has been loaded and the application is running, you can test by applying for adoption with the following request:
-
-`curl -X POST "http://{{adoption-service-host}}/adoption/applyForAdoption" -H "accept: */*" -H "Content-Type: application/json" -d "{ \"animalId\": \"e22d494c-c2be-4d32-bceb-ec675fd5540a\", \"email\": \"harvey@gmail.com\", \"kidsUnder16\": false, \"occupation\": \"Engineer\", \"ownOtherAnimals\": true, \"residency\": \"HOUSE\", \"squareFootageOfHome\": 2000, \"username\": \"Dog-Dad\"}"`
-
-To test that the email notification was delivered successfully (assuming the response was a 200) do the following:
-`oc port-forward <email-service-pod-name> <arbitrary-port>:80`
-
-Then, open a browser and go to `http://localhost:<arbitrary-port> and you should see your email`
\ No newline at end of file
+    Please be sure that the project is defined as member in the SMMR.
```

Comment:
Can we add an example of the command to verify this?

Reply:
I added a new step to make it more clear